### DISCUSSION TITLE
В 1.4.17 пропала возможность переименовать папку manager

### DISCUSSION THREAD

**Discussion by netcrawlerua** [evolution-cms/evolution#2088] [General]
manager прописан в файлах. Можно ли как-то вернуть возможность переименования?

**Comment by Dmi3yy** [comment]
В каких файлах ? Ничего не убирали

**Reply by netcrawlerua** [reply]
Извиняюсь, не четко описал проблему. Раньше можно было без проблем переименовать папку manager в любую другую (внеся соответствующую правку в файл siteManager.php). Теперь после этого выскакивает ошибка, которая на скриншоте. Также на скриншоте одно из изменений, которое к этой ошибке приводит.

<a href="https://ibb.co/2W4zdqc"><img src="https://i.ibb.co/TY3QtcP/bug.jpg" alt="bug" border="0"></a>

**Reply by Dmi3yy** [reply]
ага понял, надо будет в след релизе исправить там надо вместо manager прописать константу
